README.md

```diff
@@ -28,3 +28,64 @@ async def main():
 
 asyncio.run(main())
 ```
+
+## Development
+
+Example script for Linux how to run everything.
+
+```sh
+# Checkout the repository
+git clone https://github.com/CorantGmbH/aioairq
+cd aioairq
+
+# Create a virtual environment
+# Hints:
+# - Ensure you have Python 3.9 or above.
+#   If "python3" does not work for you or points to an older "python3.8",
+#   you may need "python3.9" or similar instead to use the correct version.
+# - You might get an error message, if you haven't installed the python3-venv module.
+#   Follow the instructions of the message.
```

Comment:
Sorry for bike-shedding. I find this a bit overly specific (in particular to the ubuntu release, which caused these hiccups in your workflow).

```suggestion
# Below we assume that your system has python of version 3.9 or higher installed and accessible via `python3`. 
# - If this command is not found or references an older version, consult your distribution.
# - Depending on the distribution, you may need to install an additional package, e.g. `python3-venv`.
```